.claude/skills/preflight/tests/judges/permission_errors.py

```
@@ -0,0 +1,46 @@
+def get_judges():
+    @scorer(name="permission-errors")
+    def permission_errors(trace) -> Feedback:
+        tool_spans = trace.search_spans(span_type=SpanType.TOOL)
+        errors = []
+
+        for span in tool_spans:
+            outputs = span.outputs or {}
+            output_str = str(outputs).lower()
+
+            if span.status and span.status.status_code and "ERROR" in str(span.status.status_code):
+                if any(p in output_str for p in PERMISSION_PATTERNS):
```

Ruff (0.15.15)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)